packages/kit-bg/src/desktopApis/DesktopApiAppUpdate.ts

```
@@ -22,6 +22,7 @@ import { EServiceEndpointEnum } from '@onekeyhq/shared/types/endpoint';
 
 import type { IDesktopApi } from './base/types';
 import type { UpdateCheckResult } from 'electron-updater';
+import { clearWindowProgressBar, updateWindowProgressBar } from '@onekeyhq/desktop/app/windowProgressBar';
```

ESLint

[error] Replace `·clearWindowProgressBar,·updateWindowProgressBar·` with `⏎··clearWindowProgressBar,⏎··updateWindowProgressBar,⏎`

(prettier/prettier)